### ISSUE TITLE
Is there a way to have access to a Charm++ version or development branch that has HIP support for AMD GPU use?

### ISSUE THREAD

**Issue by Daniel-Tichy** [charmplusplus/charm#3861]
I tried to run this 

 ./build charm++ hiplinux64 target=hip -j8
 
Error: triplet 'hiplinux64' is not supported.
Supported triplets are: mpi-crayshasta multicore-win-x86_64 mpi-win-x86_64 mpi-crayxe multicore-linux-i386 pami-linux-ppc64le mpi-linux-arm8 multicore-darwin-arm8 mpi-linux-i386 mpi-darwin-x86_64 mpi-darwin-arm8 multicore-linux-ppc64le netlrts-darwin-arm8 ucx-linux-ppc64le verbs-linux-ppc64le netlrts-win-x86_64 ofi-crayshasta mpi-linux-x86_64 verbs-linux-arm8 ucx-linux-x86_64 multicore-linux-x86_64 multicore-darwin-x86_64 netlrts-linux-x86_64 netlrts-darwin-x86_64 netlrts-linux-ppc64le pamilrts-linux-ppc64le mpi-crayxc mpi-linux-ppc64le netlrts-linux-arm7 gni-crayxe ofi-linux-x86_64 gni-crayxc multicore-linux-arm8 ucx-linux-arm8 verbs-linux-x86_64 multicore-linux-arm7 mpi-linux-arm7 netlrts-linux-arm8 netlrts-linux-i386

From the error message, Charm++ doesn’t recognize the target triplet hiplinux64, which means the build system doesn’t have a configuration for HIP (AMD GPU) under that name. Current stable releases of Charm++ typically do not provide an official triplet named hiplinux64. Instead, the standard build triplets focus on CPU targets (multicore, MPI, NetLRTS, etc.) or on NVIDIA GPU backends (cuda variants) rather than HIP.

In other words, Charm++ as-compiled from an official release tarball or from older Git commits likely does not support a HIP-specific triplet out of the box.

Is there a way to have access to a Charm++ version or development branch that has HIP support for AMD GPU use?
